src/wallet/spend.cpp

```diff
@@ -899,6 +899,13 @@ static util::Result<CreatedTransactionResult> CreateTransactionInternal(
     const CAmount not_input_fees = coin_selection_params.m_effective_feerate.GetFee(coin_selection_params.tx_noinputs_size);
     CAmount selection_target = recipients_sum + not_input_fees;
 
+    // This can only happen if feerate is 0, and requested destinations are value of 0 (i.e. ephemeral anchor)
```

Comment:
Having this hunk be in its own cherry-picked commit might make merging easier if bitcoin/bitcoin#27271 gets backported?